spec/services/moment_keywords_spec.rb

```diff
@@ -0,0 +1,34 @@
+describe MomentKeywords do
+  subject(:moment) {FactoryBot.build(:moment,
```

Comment:
This must be `let` as it does not refer to the tested class (`MomentKeywords`)

```suggestion
  let(:moment) {FactoryBot.build(:moment,
```